src/main/webapp/gameHelper.js

```diff
@@ -65,6 +73,32 @@ function nextLine() {
   quoteContainer.innerText = dialogueArray[dialogueRegex];
 }
 
+//loads the image upload url to the form
```

Comment:
Add to your PR description that these functions aren't new, just refactored to gameHelper.js

Reply:
there should be a space between // and load